Jenkinsfile

```diff
@@ -94,7 +94,7 @@ pipeline {
             steps {
                 dir('deploy/live') {
                     deleteDir()
-                    sh 'git clone -b asf-site https://gitbox.apache.org/repos/asf/camel-website.git .'
+                    sh 'git clone -b asf-site https://gitbox.apache.org/repos/asf/camel-website-pub.git .'
                     sh "git -c core.editor='sed -i 2,/\$(git log --skip=9 -1 --pretty=format:%h)/s/^pick/squash/' rebase -q --interactive $STOP_SQUASH_AT" // squash all but initial and last 9 commits
```

Comment:
`STOP_SQUASH_AT` needs to be updated

Reply:
I don't think we can do that until the asf-site branch is moved to the new repo and we determine if the OID is the same.... I think it should be, but I'm not sure.

Reply:
I also think we should stop squashing, and just keep all the history.  It doesn't slow down our website deploy, in fact it should be marginally quicker as no squash is needed.  If we tag the site just before we remove an outdated subproject version, we can point then people to the old documentation rather easily.